server/src/services/notes/notes.go

```diff
@@ -112,65 +113,56 @@ func (s *NoteService) Delete(ctx context.Context, body dto.NoteDeleteRequest, id
 		Note:  &note,
 	}
 
-	votes, vErr := s.database.GetVotes(voteFilter)
-	if vErr != nil {
-		logger.Get().Errorw("unable to retrieve votes for a note delete", "err", vErr)
+	votes, err := s.database.GetVotes(voteFilter)
+	if err != nil {
+		log.Errorw("unable to retrieve votes for a note delete", "err", err)
 	}
 
-	err := s.database.DeleteNote(user, board, id, body.DeleteStack)
+	err = s.database.DeleteNote(user, board, id, body.DeleteStack)
 	if err != nil {
+		log.Errorw("unable to delete note", "note", body, "err", err)
 		return err
 	}
 
 	s.DeletedNote(user, board, note, votes, body.DeleteStack)
-
 	return err
 }
 
 func (s *NoteService) UpdatedNotes(board uuid.UUID) {
-	notes, dbErr := s.database.GetNotes(board)
-	if dbErr != nil {
-		logger.Get().Errorw("unable to retrieve notes in UpdatedNotes call", "err", dbErr)
+	notes, err := s.database.GetNotes(board)
+	if err != nil {
+		logger.Get().Errorw("unable to retrieve notes in UpdatedNotes call", "boardID", board, "err", err)
```

Comment:
What's the difference between `logger.Get()` and `logger.FromContext()`?

Reply:
Since we store the logger in go's context, we are able to automatically log data that was afterwards added to the context.
An example would be if we get a HTTP request we also store data like request-id inside the context.
`{"level":"info","ts":...,"caller":"logger/logger.go:84","msg":"Served","requestID":"microwave/V3oqqt2ri5-000004","method":"POST","proto":"HTTP/1.1","address":"\"http://localhost:8080/boards\"","lat":0.010658187,"status":201,"size":278}
`